pom.xml

```diff
@@ -121,24 +121,24 @@
       See also: the surefire plugin section and the profiles section.-->
     <SKIP_INTEGRATION_TESTS>true</SKIP_INTEGRATION_TESTS>
     <!-- Configuration for unit/integration tests section 1 of 3 (properties) ENDS HERE.-->
-    <avro.version>1.9.2</avro.version>
-    <parquet.version>1.11.1</parquet.version>
-    <helix.version>0.9.8</helix.version>
-    <zkclient.version>0.7</zkclient.version>
-    <jackson.version>2.10.0</jackson.version>
-    <zookeeper.version>3.5.8</zookeeper.version>
+    <avro.version>1.11.0</avro.version>
+    <parquet.version>1.12.3</parquet.version>
+    <helix.version>1.0.4</helix.version>
+    <zkclient.version>0.11</zkclient.version>
+    <jackson.version>2.13.3</jackson.version>
```

Comment:
Is there a way we can use 2.12.7 which doesn't have vulnerabilities. I saw some things breaking at runtime after 2.13 upgrade